adafruit_button/button_base.py

```diff
@@ -89,14 +91,18 @@ def label(self, newtext):
 
         if not self._label_font:
             raise RuntimeError("Please provide label font")
-        self._label = Label(self._label_font, text=newtext)
-        dims = self._label.bounding_box
+        self._label = Label(self._label_font, text=newtext, scale=self._label_scale)
+        dims = list(self._label.bounding_box)
+        dims[2] *= self._label.scale
+        dims[3] *= self._label.scale
```

Comment:
I'm not in love with having to do this with the bounding box dimensions. As I mentioned in the description, I'm not sure if it was intentional or not that the bounding box for `Label` isn't scaled, but since that's a separate repository, I thought I'd just work with what we have. I'm open to suggestions here.

Reply:
I do think the approach you've gone for here is fine. It may be worth considering changing in Label in the long term, but there could be reasons in there why it's like this, I don't know for certain.